fastcrypto/Cargo.toml

```diff
@@ -64,11 +64,12 @@ name = "encoding"
 harness = false
 
 [features]
-default = ["secure"]
+default = ["secure", "threads-bls_12_377"]
 secure = []
 copy_key = []
 unsecure_schemes = ["dep:twox-hash"]
 no-threads-blst = ["blst/no-threads"]
+threads-bls_12_377 = ["celo-bls/parallel", "ark-ec/parallel", "ark-ff/parallel", "ark-std/parallel"]
```

Comment:
It's funny that the 2 BLS projects take a completely different approach re default mode for parallelism :) :)
- BLST... hey we automatically do, please expose `no-threads-blst` if you don't want it.
- celo-bls...  our default behavior is to be sequential, but you can enable the `parallel` feature (you need to enable a few features for that in our different crates)

Reply:
> celo-bls... our default behavior is to be sequential, but you can enable the parallel feature (you need to enable a few features for that in our different crates)

That's actually our crate, and the default behavior is parallelism https://github.com/celo-org/celo-bls-snark-rs/commit/78f379fe4b16348999a81f57c231102b0913224e